content/copilot/index.md

```diff
@@ -49,12 +49,29 @@ Here's how to get started with the Docker Extension for GitHub Copilot.
 Before you start, ensure that:
 
 - You have been granted access to GitHub's [limited public beta program](https://github.blog/changelog/2024-05-21-copilot-extensions-now-in-limited-public-beta/).
-- The [Docker for GitHub Copilot](https://github.com/marketplace/docker-for-github-copilot)
-  app is installed in your GitHub organization.
+- You're signed in to your GitHub account on GitHub.com.
```

Comment:
Do you think it's worth adding a note about how you'll need to enable the  Copilot extensions policy in the org per https://docs.github.com/en/copilot/using-github-copilot/using-extensions-to-integrate-external-tools-with-copilot-chat#about-using-github-copilot-extensions